plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -63,11 +63,14 @@ const registerVault = async (whatLog, plasmaFramework, ethVault, key, gnosisMult
 
 const paymentExitGameInit = async (whatLog, paymentExitGame, gnosisMultisigAbi, gnosisMultisigAddress, deployerAddress) => {
     const paymentExitGameInitCall = web3.eth.abi.encodeFunctionCall(paymentExitGame.abi.find(o => o.name === 'init'), []);
+    console.log(`paymentExitGameInitCall: ${util.inspect(paymentExitGameInitCall, { showHidden: false, depth: null })}`);
     const gnosisPaymentExitGameInit = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [paymentExitGame.address, 0, paymentExitGameInitCall]);
-    const estimateGas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisPaymentExitGameInit });
-    const gas = estimateGas * 2;
-    console.log(`The amount of gas used for ${whatLog}: ${gas}`);
+    console.log(`gnosisPaymentExitGameInit: ${util.inspect(gnosisPaymentExitGameInit, { showHidden: false, depth: null })}`);
+    // const estimateGas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisPaymentExitGameInit });
```

Comment:
Maybe delete all the commented out code and add a comment about how we're using 3000000 because estimateGas isn't reliable